tools/data_integrity/score_instrument.php

```diff
@@ -0,0 +1,241 @@
+<?php
+/**
+ * This tool scores any registered instrument that was built using the
+ * NDB_BVL_Instrument class and that has a working score() method.
+ * The command line arguments need to contain a valid test_name, a 'all' or 'one'
+ * option, a CandID and a SessionID.
+ *
+ * The 'all' option scores all existing records of an instrument if and only if
+ *  - They belong to an active timepoint
+ *  - The data-entry status of the timepoint is set to `complete`
+ *  - The administration of the timepoint is NOT be `none`
```

Comment:
@ridz1208 sorry. One last comment and good to go on my end. Remove "be" between NOT and `none`.